packages/wallets/src/features/cashier/screens/WithdrawalErrorScreen/WithdrawalErrorScreen.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { TSocketError } from '@deriv/api-v2/types';
+import { WalletsErrorScreen } from '../../../../components';
+
+type TProps = {
+    error: TSocketError<'cashier'>['error'];
+    resetError: VoidFunction;
+    setResendEmail: React.Dispatch<React.SetStateAction<boolean>>;
+};
+
+const WithdrawalErrorScreen: React.FC<TProps> = ({ error, resetError, setResendEmail }) => {
+    if (error.code === 'InvalidToken') {
```

Comment:
v. minor / optional: potentially create enum for token codes for specific endpoint? hardcoding strings generally should be avoided

Reply:
sorted

Reply:
![Screenshot 2024-05-10 at 11 25 11](https://github.com/binary-com/deriv-app/assets/103181646/ff05a86e-4b73-41ab-92c7-bd532d878fb6)